### DISCUSSION TITLE
Make the class UserInfo use available on the flutter front.

### DISCUSSION THREAD

**Discussion by Tristannos** [serverpod/serverpod#801] [Q&A] [answered]
I would like to use the UserInfo model in the flutter front because then i could update the UserInfo live with my stream.

<img width="414" alt="SCR-20230329-kl0" src="https://user-images.githubusercontent.com/55278228/228540286-a73e0e1d-c746-49d1-b7da-825a66864261.png">

And The user model is not in the imports of the generated files. I have to manualy import them every time.

<img width="402" alt="SCR-20230329-klx" src="https://user-images.githubusercontent.com/55278228/228540568-8c3f5e7e-c94d-4abe-9920-cdf9901f2881.png">

And why can't we just change the UserInfo model just like any other model? 
It's just such a hassle to create a extra model to add additional information and always getting 2 models when returning user information.

**Comment by vlidholt** [comment, marked as answer]
You will need to import the `UserInfo` object from the `serverpod_auth_client` package in your Flutter app.
